client/src/app/shared/components/data-table/data-table.component.ts

```diff
@@ -67,4 +82,26 @@ export class DataTableComponent implements OnInit {
 
     return true;
   }
+
+  private fetchData(params) {
+    if (this.endpoint && this.entityType) {
+      this.ingestService.get(this.endpoint, {params: params}).pipe(
```

Comment:
Is using generic get here actually making it that generic? We're also hardcoding entityType so this can only be used for endpoints that return entities.

Reply:
yeah, i wanted to make this data-table generic to display any result conforming to that Ingest Core's response format. However, I had to filter it the specific metadata types to only display the information inside content property. I'll improve this later.